LEGO1/library_msvc.h

```diff
@@ -783,4 +783,11 @@
 // GLOBAL: BETA10 0x101faf78
 // __app_type
 
+// GLOBAL: LEGO1 0x100db6e0
+// GUID_SysKeyboard
+
+// Cannot be handled right now due to anonymous pointer in struct
+// // GLOBAL: LEGO1 0x10098f80
+// c_dfDIKeyboard
```

Comment:
I didn't have a good idea how to solve this one. It will fail `datacmp` because it points to a structure that contains a pointer. However, the pointee of that structure does not have a name/debug symbol.

Reply:
We would likely need an annotation like
```
// GLOBAL: BETA10 0x10097f80
// c_dfDIKeyboard.rgodf
```
but we don't have anything like at right now, as far as I am aware.